airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/BasicAcceptanceTests.java

```diff
@@ -532,7 +532,7 @@ public void testDeleteConnection() throws Exception {
     testHarness.removeConnection(connectionId);
 
     LOGGER.info("Waiting for connection to be deleted...");
-    Thread.sleep(500);
+    Thread.sleep(5000);
```

Comment:
Is this making the test being flaky? I haven't experienced it myself.

Reply:
I've seen this several times in CI on other PRs as well - with the current 500ms sleep, temporal sometimes doesn't have enough time to actually delete the connection and this test fails. 5 seconds seems to be enough time for it to pass consistently, and adding a few more seconds to these acceptance tests won't hurt much